pandas/core/computation/align.py

```diff
@@ -36,8 +39,8 @@ def _align_core_single_unary_op(
 
 
 def _zip_axes_from_type(
-    typ: Type[FrameOrSeries], new_axes: Sequence[int]
-) -> Dict[str, int]:
+    typ: Type[FrameOrSeries], new_axes: List["Index"]
```

Comment:
can you keep this as Sequence. The only requirement on new_axes in the implementation within this function is that it has a \_\_getitem__ method.
(https://docs.python.org/3/library/collections.abc.html)

from https://github.com/python/typeshed/blob/master/CONTRIBUTING.md#conventions (Some further tips for good type hints:)

> avoid invariant collection types (List, Dict) in argument positions, in favor of covariant types like Mapping or Sequence;

Reply:
Done.